src/main/webapp/login.jsp

```diff
@@ -1,32 +1,40 @@
-<%@ page contentType="application/xhtml+xml; charset=UTF-8" pageEncoding="UTF-8" %>                    
+<%@ page contentType="application/xhtml+xml; charset=UTF-8" pageEncoding="UTF-8" %>
 <%@ include file="/WEB-INF/jspf/taglibs.jspf" %>
 <%@ include file="/WEB-INF/jspf/header.jspf" %>
 
 
-<h1>OpenSRP</h1>
-
-<div id="content">
+<div styleClass="oauth-container-div text-center" id="content">
+  <form styleClass="oauth" id="loginForm" name="loginForm" action="<c:url value='/login.do'/>" method="post">
+    <img styleClass="mb-4 opensrplogo" src="images/opensrplogo.png" alt="opensrp logo" />
+    <h1 styleClass="h3 mb-3 font-weight-normal">Please Log in</h1>
     <c:if test="${not empty param.authentication_error}">
-        <h1>Error!</h1>
+      <div styleClass="alert alert-danger" role="alert">
+        <p styleClass="h3 mb-3 font-weight-normal">Error!</p>
+
+        <p>Your login attempt was not successful.</p>
+      </div>
 
-        <p styleclass="error">Your login attempt was not successful.</p>
     </c:if>
     <c:if test="${not empty param.authorization_error}">
-        <h1>Error!!</h1>
+      <div styleClass="alert alert-danger" role="alert">
+        <p styleClass="h3 mb-3 font-weight-normal">Error!</p>
+
+        <p>You are not permitted to access that resource.</p>
+      </div>
+
 
-        <p styleclass="error">You are not permitted to access that resource.</p>
     </c:if>
 
+    <label for="username" styleClass="sr-only">Username</label>
+    <input type="text" id="username" name='j_username' styleClass="form-control" placeholder="Username" required autofocus />
 
-    <h2>Login</h2>
+    <label for="password" styleClass="sr-only">Password</label>
```

Comment:
Please wrap this field in a `form-group` classed div.  Look at the Bootstrap4 example forms to see how this is done.